Delegation program urgent updates (#771)
o1-labs/docs2: docs/node-operators/foundation-delegation-program.mdx

### What happens after I receive a delegation?

#### Payout Procedure

> As per the Mina Foundation Delegation policy, there is a mandatory requirement on delegatees to use the latest version of [thew payout script](https://github.com/jrwashburn/mina-pool-payout) to manage the return of rewards.

Reviewer: ```suggestion
As per the Mina Foundation Delegation policy, there is a mandatory requirement on delegatees to use the latest version of the [payout script](https://github.com/jrwashburn/mina-pool-payout) to manage the return of rewards.
```

Mina Foundation has collaborated on updating the well-known payout script (by Jonathan Washburn and Gareth Davies) to reflect and support its Delegation Program Policy requirements

> This payout script includes

Reviewer: ```suggestion
This payout script includes:
```

- Mapping of delegating wallet addresses with the correct return wallet address
- Calculation of the Reward amounts and paybacks of the remaining rewards to the delegators.
- Calculation of the supercharged rewards to be burned and sent to the burn address.
- Specific memo which includes the md5 hash of Pool’s Public Key (MD5 hash)

> To see more, please check the  [https://github.com/jrwashburn/mina-pool-payout/blob/main/README.md](README) where you can review the script and supporting README

Reviewer: ```suggestion
For more details and to review the script, see the [README](https://github.com/jrwashburn/mina-pool-payout/blob/main/README.md) file in the `mina-pool-payout` repository.
```
